### PR TITLE
Push 16 to 25% of the instances with 15 on the stable channel

### PR DESCRIPTION
**Pull request by jospoortvliet**
WIP

### CODE REVIEW

**Comment by jospoortvliet** [comment]
Ok, I hope I got it right now... waiting for tests to run.

**Comment by MorrisJobke** [comment]
Also this is not correct. It needs to be inside the 15 where you changed it to 75.

File: config/config.php [hunk 1]
```diff
@@ -310,6 +310,20 @@
...
+				'web' => 'https://docs.nextcloud.com/server/16/admin_manual/maintenance/upgrade.html',
+				'eol' => false,
+				'minPHPVersion' => '7.1',
+				'signature' => 'ZQgU/ITrJK9qjby4BowlWnlpTdG7izvqiwrjwm7iTTN8eMlCPP0np9e5znnO9pmR
+MS5Fg8wD3xsbudZk89x2oY36xCF/iB6GRrUh5mB8jDeGLuDt03HkEjd7QhENscVD
+WKKLCydVsFvAyZKkvoloRW6IcN3JxGKdp4dRYzmjftc6lnisX0LiSaOKpc7OOShI
++udimp0LXmz3qudO/ydY+kofuTdax76nY93Mkxv+Hg8ZuWb/fC9Bo203vBziLzNL
+vMoAbVrciXWFHtyxpa4VrTzxa21mLnkIbnCxUWHbGD8RADoKcgdtiIdRraZrbiAO
+5RiPj0P99h2+hYL1XcGcAQ==',
+			],
```
Comment (MorrisJobke) [inline config/config.php]: Typically this config has the newest version at the top and the smaller ones at the bottom. Could you change this as well? Or let me quickly do it.

**Comment by MorrisJobke** [comment]
I also added a negative test - just to be sure.

**Comment by jospoortvliet** [comment]
thanks a lot morris!

**Pull request merged**